components/01-atoms/divider/divider.stories.js

```diff
@@ -22,12 +22,12 @@ export default {
 };
 
 export const Dividers = ({ position, thickness, dividerColor }) => `
+  ${dividerTwig({ divider__thickness: 'hairline' })}<br />
```

Comment:
At first I thought this was weird you were skipping 3 and 5. Then I realized after looking at the token definitions that this was a multiplier. 

Hairline does seem like the weird outlier here and I'm wonder if there is a better way to address this? Possible ideas:
- `thin` (easier to type)
- Refactor `hairline` to be 1 and adjust the others accordingly.
- `$size-thickness-0-5` given it is half the size of `$size-thickness-1`

Reply:
I'm not going to die on this hill though.

Reply:
Yeah, I had the same concerns and talked with the designers about it. We tried to brainstorm some different ways, but couldn't come up with anything better. So we'll stick to this for now